recipes/dipy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.11.0" %}
+
+package:
+  name: dipy
+  version: {{ version }}
+
+source:
+  fn: dipy-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/d/dipy/dipy-{{ version }}.tar.gz
+  md5: 7723e8994d31e910d47b7a0cf064a3d8
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - nibabel
+    - cython
+    - gcc  # [not win]
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - nibabel
+    - scikit-learn
+    - matplotlib
+    - libgcc  # [not win]
+    - vtk  # [py27]
+test:
+  imports:
+    - dipy
+
+about:
+  home: https://nipy.org/dipy
+  license:  3-clause BSD license
+  license_file: LICENSE
+  summary: Diffusion MR Imaging in Python
+
+extra:
+  recipe-maintainers:
+    - arokem
+    - garyfallidis
```

Comment:
I'm sure you guys have already discussed this, but please let us know whether it is ok to have you as a maintainer on the `dipy` recipe at [conda-forge]( http://conda-forge.github.io/ ), @garyfallidis.